docs/static_site/src/pages/get_started/windows_setup.md

```diff
@@ -137,126 +137,46 @@ Check the chart below for other options or refer to [PyPI for other MXNet pip pa
+### Optional step:
 
-&nbsp;
-Next, we install ```graphviz``` library that we use for visualizing network graphs you build on MXNet. We will also install [Jupyter Notebook](http://jupyter.readthedocs.io/)  used for running MXNet tutorials and examples.
+install ```graphviz``` library that we use for visualizing network graphs you build on MXNet. We will also install [Jupyter Notebook](http://jupyter.readthedocs.io/)  used for running MXNet tutorials and examples.
 - Install ```graphviz``` by downloading MSI installer from [Graphviz Download Page](https://graphviz.gitlab.io/_pages/Download/Download_windows.html).
 **Note** Make sure to add graphviz executable path to PATH environment variable. Refer [here for more details](http://stackoverflow.com/questions/35064304/runtimeerror-make-sure-the-graphviz-executables-are-on-your-systems-path-aft)
 - Install ```Jupyter``` by installing [Anaconda for Python 2.7](https://www.anaconda.com/download/)
 **Note** Do not install Anaconda for Python 3.5. MXNet has a few compatibility issues with Python 3.5.
 
-We have installed MXNet core library. Next, we will install MXNet interface package for programming language of your choice:
-- [Python](#install-the-mxnet-package-for-python)
-- [R](#install-the-mxnet-package-for-r)
-- [Julia](#install-the-mxnet-package-for-julia)
-- **Scala** is not yet available for Windows
 
 ## Install the MXNet Package for Python
 
-These steps are required after building from source. If you already installed MXNet by using pip, you do not need to do these steps to use MXNet with Python.
-
-1. Install ```Python``` using windows installer available [here](https://www.python.org/downloads/release/python-2712/).
-2. Install ```Numpy``` using windows installer available [here](https://scipy.org/index.html).
-3. Start ```cmd``` and create a folder named ```common```(```mkdir C:\common```)
-4. Download the [mingw64_dll.zip](https://sourceforge.net/projects/openblas/files/v0.2.12/mingw64_dll.zip/download), unzip and copy three libraries (.dll files) that openblas.dll depends on to ```C:\common```.
-5. Copy the required .dll file to ```C:\common``` and make sure following libraries (.dll files) in the folder.
-```
-libgcc_s_seh-1.dll (in mingw64_dll)
-libgfortran-3.dll (in mingw64_dll)
-libquadmath-0.dll (in mingw64_dll)
-libopenblas.dll (in OpenBlas folder you download)
-opencv_world341.dll (in OpenCV folder you download)
-```
-6. Add ```C:\common``` to Environment Variables.
- * Type ```control sysdm.cpl``` on ```cmp```
- * Select the **Advanced tab** and click **Environment Variables**
- * Double click the **Path** and click **New**
- * Add ```C:\common``` and click OK
-7. Use setup.py to install the package.
+Use setup.py to install the package.
 ```bash
     # Assuming you are in root mxnet source code folder
-    cd python
-    python setup.py install
+    pip install --upgrade --force-reinstall -e python
```

Comment:
This gives an error:
```
pip install --upgrade --force-reinstall -e python
ERROR: python is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with svn+, git+, hg+, or bzr+).
```

Reply:
Works like a charm for me, I have tried again.